test/typecheck/BUILD

```diff
@@ -44,4 +45,5 @@ go_test(
     name = "go_default_test",
     srcs = ["main_test.go"],
     embed = [":go_default_library"],
+    deps = ["//vendor/golang.org/x/tools/go/packages:go_default_library"],
```

Comment:
this needs a `data` entry with the testdata, I think roughly `data = ["//test/typecheck/testdata:all-srcs"]` though that target doesn't appear to exist ...